pkgs/servers/fishnet/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, stdenv
+, rustPlatform
+, callPackage
+, fetchFromGitHub }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "fishnet";
+  version = "2.2.3";
+
+  src = fetchFromGitHub {
+    owner = "niklasf";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "159fwjy70n6lvnhdwv65azgi03r5qcc2m2zpzgz0k3r6cy06faxj";
+  };
+
+  assets = callPackage ./assets.nix { };
```

Comment:
Please do not use callPackage in callPackage. I think creating the derivation in a let in would preferable.